rest-api/edit-conversation-image-title/server/server.js

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
+import cors from 'cors';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const app = express();
+app.use(cors())
+app.use(bodyParser.json())
+
+async function editImageOrTitle(conversationId, imageURL, conversationTitle) {
+  
+  const data = conversationTitle === "" ? { photoUrl: imageURL } : imageURL === "" ? { subject: conversationTitle } : { subject: conversationTitle, photoUrl: imageURL };
+
+  const talkJSURL = `${process.env.TALKJS_URL}/${process.env.APP_ID}/conversations/${conversationId}`;
```

Comment:
Would suggest making a template `.env` file with placeholder values (or just adding placeholders as constants in the code)

Reply:
I have added that to the repo and in the instructions as well.